src/components/pages/login/index.js

```diff
@@ -2,15 +2,23 @@ import { Cookies } from 'react-cookie';
 
 const Index = () => null;
 
-export const getServerSideProps = ({ req, params }) => {
+export const getServerSideProps = ({ req }) => {
   const cookies = new Cookies(req?.headers?.cookie);
 
   const language = cookies.get('udb-language') ?? 'nl';
 
+  const url = new URL(`http://${req.headers.host}${req.url}`);
```

Comment:
Shouldn't this be https:// ?
```suggestion
  const url = new URL(`https://${req.headers.host}${req.url}`);
```